### DISCUSSION TITLE
Coupling Sandbox with ALM

### DISCUSSION THREAD

**Discussion by dineshkumarreddyg36** [xcompact3d/Incompact3d#155] [Q&A] [answered]
Hi,
I'm trying to study the effect of a upstream deflector on the performance of a turbine farm for which I'm using the Sandbox.
But, in Sandbox all the parameters are non dimensionalized and in ALM everything is Dimensionalized. I've generated the deflector using the Xcompact3d-toolbox and for eg. if I want to give a flow velocity of 8.5 m/s inside the domain how can I achieve that because the values of u1 and u2 that the Sandbox code accepts is 1 and 2. Can someone explain this?
Thank you.

**Comment by slaizet** [comment, marked as answer]
Hi,
You will either have to do everything dimensionalised or non-dimensionalised (so you either need to modify the sandbox or the input file). In sort, the sandbox was designed for non-dimensionalised simulations, but the wind farm simulator was designed for dimensionalised simulations. Not ideal but not a big problem :-)
Best,
Sylvain

**Reply by dineshkumarreddyg36** [reply]
@slaizet Can I do it in this way?
For Eg. for U=8.5m/s I've to give the value of **1/nu** in input file as 67567. So now making the U=1 the value of **1/nu** becomes 5.74324e5 in order to maintain the same flow Reynolds Number and also **Uref** in **input_test.turb** file as 1. So, basically playing only with the value of 1/nu and keeping everything else same.
Thank You.

**Reply by slaizet** [reply]
You also need to deal with a reference length scale (very likely to be the diameter of the turbine) and with a reference density (very likely to be the density of air). So as an example, you will have to change the dimensions of the simulation (not in km but in diameter). You can normalise all relevant quantities with a reference velocity, a reference density and a reference length scale.

Best,

Sylvain
